docs/topics/Terms-of-Use.md

```diff
@@ -0,0 +1,23 @@
+# Terms of Use
+
+By using the o!TR API, you agree to abide by these terms of use. The o!TR API is provided publicly, for free, for all users by the osu! Tournament Rating Team ("o!TR Team", "us", "our").
+
+## Scraping
+
+The API must not be used to scrape information. Scraping data, such as tournaments, matches, etc. is forbidden and will result in a permanent revocation of API access. Please see the [data](Data.md) page for more information on conveniently importing our dataset.
+
+## Accreditation
+
+Data which is sourced from us should be credited properly. Misleading users to believe data created by us is provided by another source, or failing to credit this information as being created by us, is not allowed.
+
+> For example, informing users of your application that information is provided by osu! officially is not allowed because it is not true.
+> 
+{style="warning"}
+
+## Abuse
+
+Users who abuse this service will have their API access revoked. For example, repeatedly encountering 429 errors will result in access revocation.
+
+### Rate Limit Increases
+
+If you wish to receive an increase to the API rate limit of your application, please [contact us](Contact.md) with your use case.
```

Comment:
is it worth writing here what the default rate limits are?

Reply:
I think that info is better suited under the actual spec section (coming at some point in the future). That way we don't have to maintain it in two places.